README.md

```diff
@@ -26,11 +26,9 @@ These steps come from the [extension dev page](https://developer.mozilla.org/en-
 
 ### Setup specific to this addon
 1. If you're hacking on the [iframe](https://github.com/mozilla/universal-search-content), then configure your profile to use a local copy:
-  - Set these two prefs in about:config:
-  - `services.universalSearch.frameURL`: the complete iframe URL
-    - default is 'https://d1fnkpeapwua2i.cloudfront.net/index.html'
-  - `services.universalSearch.baseURL`: the iframe's base URL
-    - default is 'https://d1fnkpeapwua2i.cloudfront.net'
+  - Run `gulp gen-prefs`
+    - creates a file named `users.js`
+    - move this file to your Firfox profile directory, e.g. `/path/to/ff/Profiles/6h6ygzlo.addon-dev/users.js`
```

Comment:
Not sure if it is worth pointing users to something like https://support.mozilla.org/en-US/kb/profiles-where-firefox-stores-user-data (on the off chance they, like me, forget where their profile directory was). #iamnotasmartman